app/src/main/java/com/andrew/tasky/agenda/data/event/EventRepositoryImpl.kt

```diff
@@ -30,25 +35,38 @@ class EventRepositoryImpl @Inject constructor(
                     ),
                 photoData = event.photos.filterIsInstance<EventPhoto.Local>().mapIndexed {
                     index, eventPhoto ->
+                    val imageByte = withContext(Dispatchers.IO) {
+                        appContext.contentResolver.openInputStream(eventPhoto.uri).use {
+                            it!!.readBytes()
+                        }
+                    }
+                    val compressedImageByte = compressByteArrayToSize(imageByte, 1000000)
```

Comment:
I'd create a separate class for reading bytes from a content uri to follow the single source of truth principle. That allows you to easily reuse this logic and it's easier to find and change. I created a `UriByteConverter` class or so for that